infrastructure-aliroot-releases.markdown

```diff
@@ -0,0 +1,90 @@
+---
+title: Make AliRoot releases
+layout: main
+categories: infrastructure
+---
+
+Versioning scheme
+=================
+The current release process of AliRoot was agreed with the Data Preparation group. It has two main
+release models:
+
+* **Validated releases.** AliRoot tags are in the format **v5-08-13**. The minor number is increased
+  and no letters are added after the name. Tags are from the master branch. The DPG validation is
+  available [here](https://alijenkins.cern.ch/job/AliRootDPGValidation/)
+
+* **Releases not requiring validation.** AliRoot tags are in the format **v5-08-13abc**, note the
```

Comment:
This is fine, however it is IMHO confusing for the case in which the change is actually a major refactoring, breaking compatibility with all the previous releases but still not needing release validation (e.g. because what we really want is to be able to test PRs with it). How do we handle that case?

Reply:
The DPG determines whether a validation is necessary or not.

Note that this PR is to _document_ the current workflow, not to _discuss_ it 😄 any change must be discussed to the DPG (and we are not changing the _current_ model since Run2 is over, there are margins of improvement for Run3 but I won't be the one handling them).